### DISCUSSION TITLE
Tailwind 2.2.0 features

### DISCUSSION THREAD

**Discussion by srid** [tw-in-js/twind#191] [Ideas]
Tailwind 2.2.0 introduced a few features like `selection:bg-pink-200`,

https://github.com/tailwindlabs/tailwindcss/releases/tag/v2.2.0

Unfortunately they work only in JIT mode (not say in CDN). I think this is one place where twind shim can shine. 

Will twind support these new features?

**Comment by sastan** [comment]
Yes. I have waited for the final feature set / spec and will start to implement these in the next days.

**Reply by sastan** [reply]
Some of the new pseudo selector variants like `selection` can already be used using the double colon syntax: `selection::bg-pink-200`. They will be available as single colon variants.
